core/new-gui/src/app/workspace/component/left-panel/left-panel.component.ts

```diff
@@ -1,61 +1,37 @@
 import { Component, OnInit } from "@angular/core";
 import { UntilDestroy, untilDestroyed } from "@ngneat/until-destroy";
-import { DynamicComponentConfig } from "../../../common/type/dynamic-component-config";
-import { OperatorMenuFrameComponent } from "./operator-menu-frame/operator-menu-frame.component";
+import { OperatorMenuComponent } from "./operator-menu/operator-menu.component";
 import { VersionsFrameComponent } from "./versions-frame/versions-frame.component";
+import { ComponentType } from "@angular/cdk/overlay";
 import {
   OPEN_VERSIONS_FRAME_EVENT,
   WorkflowVersionService,
 } from "../../../dashboard/user/service/workflow-version/workflow-version.service";
 
-export type LeftFrameComponent = OperatorMenuFrameComponent | VersionsFrameComponent;
-export type LeftFrameComponentConfig = DynamicComponentConfig<LeftFrameComponent>;
```

Comment:
lets keep the specific definitions for type check purpose. they can be modified to fit the new dynamic method.

Reply:
Fixed.